.semaphore/semaphore.yml

```diff
@@ -83,10 +83,22 @@ blocks:
             # Check shell scripts for issues.
             - "shellcheck package/*.sh package/deb-postinst"
 
-            # Run build and tests in Nix, and push the result to Cachix.
+            # Print working directory for debugging purposes
+            - "pwd"
+
```

Comment:
Is this still necessary?

Reply:
Not strictly, but I think it is useful to record the working directory since we are creating the `nix-store-location` file there.